.github/workflows/notion.yml

```
@@ -0,0 +1,69 @@
+name: Update Notion Task Status
+
+on:
+  pull_request:
+    types: [opened, closed]
+
+jobs:
+  update-notion:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)